steps/terraform/set-input-variables-azure.yml

```
@@ -48,6 +48,7 @@ steps:
                 --arg aks_network_dataplane "$NETWORK_DATAPLANE" \
                 --arg k8s_machine_type "$K8S_MACHINE_TYPE" \
                 --arg k8s_os_disk_type "$K8S_OS_DISK_TYPE" \
+                --argjson enable_apiserver_vnet_integration "$([ "$ENABLE_APISERVER_VNET_INTEGRATION" = "true" ] && echo "true" || echo "false")" \
```

Is there any issue with passing it as this instead of argjson. if we don't have any value in the variable it will take empty string which will be remove from json later and it takes false value from the json input default value.

```suggestion
                --arg enable_apiserver_vnet_integration "$ENABLE_APISERVER_VNET_INTEGRATION" ```